.golangci.yml

```diff
@@ -21,7 +21,14 @@ linters-settings:
     lines: 100
     statements: 50
   gci:
-    local-prefixes: github.com/mesosphere/konvoy-image-builder
+    sections:
+      - standard # Standard section: captures all standard packages.
+      - default # Default section: contains all imports that could not be matched to another section type.
+      - prefix(github.com/org/project) # Custom section: groups all imports with the specified Prefix.
```

Comment:
```suggestion
      - prefix(github.com/mesosphere/konvoy-image-builder) # Custom section: groups all imports with the specified Prefix.
```

Reply:
Oops sorry for copy pasta in my previous suggestion!

Reply:
Oh wait I didn't do that!

Reply:
nice catch :joy: